ClinGen/clincoded
# Connects to #917 Update quickstart doc, updated to help doc

**To test:** 

**Check hover on Question mark icons changed to general "Curation Help" for now (rather than Quick Start):**
 e.g. 

Top Nav
- ![image](https://cloud.githubusercontent.com/assets/1878194/17816739/941c1dd6-65ef-11e6-985a-454008b2cccf.png)

Select Var link on Dashboard
- ![image](https://cloud.githubusercontent.com/assets/1878194/17816744/9f81a2ea-65ef-11e6-8814-58c126f41fbe.png)



**Check updated link to help doc (clingen-variant-curation-help.pdf):**

- ![image](https://cloud.githubusercontent.com/assets/1878194/17816766/bc5fa6be-65ef-11e6-8712-fab956bdea17.png)
